src/app/treelist/treelist.component.ts

```diff
@@ -1,11 +1,11 @@
 import { Component, Input, TemplateRef, ViewChild, ViewEncapsulation } from '@angular/core';
-import { TreeNode, TREE_ACTIONS, KEYS, IActionMapping, TreeComponent } from 'angular2-tree-component';
+import { TreeNode, TREE_ACTIONS, IActionMapping, TreeComponent } from 'angular2-tree-component';
 
 // See docs: https://angular2-tree.readme.io/docs
 @Component({
   selector: 'alm-tree-list',
   template: require('./treelist.component.html'),
-  styles: [require('./treelist.component.css')],
+  styles: [require('./treelist.component.css').toString()],
```

Comment:
Almost certainly the loaders are misconfigured for .css files, as we normally use .scss. Please can you open an issue in planner and assign to me, I'll try to sort it out.